src/fred/cli/categories.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+"""
+FRED CLI - Categories Namespace.
+"""
+import json
+
+import click
+
+from .. import BaseFredAPIError
+from .._util import generate_api_kwargs
+
+
+__all__ = [
+    "categories",
+]
+
+
+@click.group()
+@click.pass_context
+def categories(ctx):
+    """
+    Categories CLI Namespace.
+    """
+    pass
+
+
+@categories.command()
+@click.option("--category-id", "-c", required=True, type=click.INT, help="Category ID.")
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_category(ctx, category_id: int, args: tuple):
+    """Get a category."""
+    try:
+        click.echo(json.dumps(ctx.obj["client"].get_category(category_id, **generate_api_kwargs(args)), indent=4))
+    except (ValueError, BaseFredAPIError) as e:
+        click.echo(click.style(e, fg="red"))
+
+
+@categories.command()
+@click.option("--category-id", "-c", required=True, type=click.INT, help="Category ID.")
+@click.argument("args", nargs=-1)
+@click.pass_context
+def get_category_children(ctx, category_id: int, args: tuple):
+    """Get the child categories."""
+    try:
+        click.echo(
+            json.dumps(ctx.obj["client"].get_category_children(category_id, **generate_api_kwargs(args)), indent=4)
+        )
+    except (ValueError, BaseFredAPIError) as e:
+        click.echo(click.style(e, fg="red"))
```

Comment:
Let's abstract this `click.echo` into `fred._util.cli`.

You can call it something like `echo_error`. While you're at it, create four new utility functions (don't forget to add them to `__all__`):

1. `echo_error` -> print red message to click console
2. `echo_warning` -> print yellow message to click console
3. `echo_success` -> print green message to click console
4. `echo_info` -> print message to click console (no color)

Reply:
Please find and replace to use these new utilities throughout the project